.github/workflows/pipeline-segment-electron-test.yml

```diff
@@ -146,6 +146,13 @@ jobs:
         sudo security authorizationdb write com.apple.trust-settings.admin allow
         cd src/electron
         ./script/codesign/generate-identity.sh
+    - name: Configure Datadog Test Visibility
+      uses: datadog/test-visibility-github-action@v2
```

Comment:
This action has the same issue as our manual implementation on appveyor, it bypasses anything that looks like a lockfile. We should add the `dd-trace` module to our `spec/package.json` dev deps and configure it from that location instead of using this action or adding it dynamically at CI time